index.html

```diff
@@ -2,32 +2,32 @@
 <html lang="en">
 
 <head>
-	<meta charset="utf-8">
-	<title>Orange Inc.</title>
-	<link rel="stylesheet" href="css/styles.css">
-	<script src="js/script.js"></script>
+  <meta charset="utf-8">
+  <title>Orange Inc.</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="js/script.js"></script>
 </head>
 
 <body>
 
-<div class="header">
-	<h1>-todos-</h1>
-	<input id="textbox" type="text" class="inputtextbox">
-	<input id="" class="submitbutton" type="submit" value="Submit" onclick="addItem()">
-</div>	
+  <div class="header">
+    <h1>-todos-</h1>
+    <input id="textbox" type="text" class="inputtextbox">
+    <input id="" class="submitbutton" type="submit" value="Submit" onclick="addItem()">
```

Comment:
Nowadays, people don't add javascript function calls to the `onclick` handler. This doesn't work well for more complex use cases. Instead, your javascript should have a reference to this input and then attach an event listener. 

Example:
```
var submitInput = document.getElementById('submit-btn')
submitInput.addEventListener('click', function (evt) { // do something })